### PR TITLE
[PayPal] Billing Address For Payer Info

### PR DESCRIPTION
**Pull request by eliasjpr**
# Summary
We at the New York Times are in need of retrieving the billing address
for a given Billing Agreement Nonce using the Vault Flow.

We would like to propose the follow changes to satisfy this need.

# Changes

- Adds Address object to enable retrieval of the PayPal Billing Address
from the backend.

# Side Effects

Can expect the Billing Address to be populated from the Billing Address
found in PayPal.

Example
Screen Shot 2020-03-19 at 11 43 17 AM

# Checklist

- [x] Added changelog entry
- [x] Ran unit tests (`mvn verify -DskipITs`)
Screen Shot 2020-03-19 at 12 05 14 PM

### CODE REVIEW

**Comment by eliasjpr** [comment]
@braebot @dan-manges I would be happy to add unit and integration tests if the changes are acceptable.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 2.107.1
```
Comment (crookedneighbor) [inline CHANGELOG.md:1]:
```suggestion
## unreleased
```

**Comment by eliasjpr** [comment]
Sounds good. Let me know if you need anything on our end.

**Comment by hollabaq86** [comment]
@eliasjpr We wanted to send a quick heads up, we're building on this work you submitted. The main change we're making is to use international address labels, so `getCity` -> `getLocality`. This way it's in parity with other Address objects returned by our API.

**Comment by eliasjpr** [comment]
Thanks for the heads up I have informed my team about this change.

**Comment by hollabaq86** [comment]
Hi @eliasjpr Happy friday! We just merged in our updates with your work to `master`, and plan to include this in the next release (which we're hoping to do next week). I'm going to go ahead and close this PR, thanks again for your contribution 😃

**Comment by crookedneighbor** [comment]
This is out in 2.109.0 https://github.com/braintree/braintree_java/commit/3cf022e01a8598c658a0c25d847f42838471c781

**Pull request closed without merging**